hera_cal/frf.py

```diff
@@ -26,22 +28,21 @@ def timeavg_waterfall(data, Navg, flags=None, nsamples=None, rephase=False, lsts
     weighted by a boolean flag array (flags) and also optionally by an Nsample array (nsample),
     such that, for a single frequency channel, the time average is constructed as
 
-    avg_data = sum( data * flag_wgt * nsample ) / sum( flag_wgt * nsample )
+    avg_data = sum( data * flag_wgt * nsample**2 ) / sum( flag_wgt * nsample**2 )
```

Comment:
Why? If the noise goes like 1/sqrt(nSamples), then inverse variance weighting would be proportional to nsamples, not nsamples**2.

Reply:
ah that's right...